db/migrate/20181211102322_add_reset_to_users.rb

```diff
@@ -0,0 +1,6 @@
+class AddResetToUsers < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
when still developing on localhost and staging, prefer to always use the original migration, avoid adding new migrations. 

After you add fields to the original migration;  you,
rails db:drop, rails db:create, rails db:migrate or rails db:setup

Reply:
ok

Reply:
done